src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ProviderHandler.java

```diff
@@ -203,15 +203,15 @@ public static void removeProviderInternal(final IStandardRequestManager manager,
      *
      * @throws IllegalArgumentException when the token is not belonging to a registered provider.
      */
-    public static ImmutableCollection<IToken> getRegisteredResolvers(final IStandardRequestManager manager, final IToken token) throws IllegalArgumentException
+    public static ImmutableCollection<IToken<?>> getRegisteredResolvers(final IStandardRequestManager manager, final IToken<?> token)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)